etc/helm/values.yaml

```diff
@@ -0,0 +1,50 @@
+# Default values for haskell-lang.
+# This is a YAML-formatted file.
+# Declare variables to be passed into your templates.
+# set fullName=$DEPLOYMENT_NAME
+# set name=$DEPLOYMENT_APP
+# set image.image=DEPLOYMENT_IMAGE
+# set image.env.approot=$APPROOT
+
+replicaCount: 1
+
+image:
+  image: deployment_image
+  pullPolicy: IfNotPresent
+  replicaCount: 2
+  minReadySeconds: 5
+  env:
+    approot: APPROOT
+  port:
+    containerPort: 9090
+
+service:
+  type: ClusterIP
+  port: 9090
+
+ingress:
+  enabled: false
```

Comment:
We'll want ingress enabled here. I'm not sure if you override it somewhere else?

Reply:
Good catch ingress should be enable